gh-actions/common/build-debian/action.yml

```diff
@@ -160,6 +164,40 @@ runs:
         path: ${{ env.SOURCE_OUTPUT_DIR }}/
         if-no-files-found: error
 
+    - name: Run lintian on sources
```

Comment:
I would be good to explain why you want to run lintian in a container and not just installing it as part of dev package. (I do see you create an user, but I’m blanking on why)

Reply:
Well, it brings a lot of dependencies in, so I don't want end up in creating false positives by being able to build a source package when the required -dev dependencies are not yet installed.

Reply:
mind adding a comment for it?

Reply:
Ok done